test/Dapr.E2E.Test.App/Startup.cs

```diff
@@ -62,9 +63,17 @@ public void ConfigureServices(IServiceCollection services)
                 // Example of registering a "PlaceOrder" workflow function
                 options.RegisterWorkflow<string, string>("PlaceOrder", implementation: async (context, input) =>
                 {
+
+                    System.Threading.Thread.Sleep(10000); // sleep for 10s to allow the terminate command to come through
```

Comment:
If we must wait a fixed amount of time, perhaps `await Task.Delay(TimeSpan.FromSeconds(10))`?